Lesson 4/dmitriy.pereiaslovec/nginx.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+progr_name='nginx'
+
+echo "Check for current user rights";
+if [[ $(id -u) -ne 0 ]]
+  then
+    echo "Please run as root"
+    exit 0;
+  else 
+    echo "Start executing tasks" 
+fi
+echo ""
+echo ""
+echo "------------------------"
+echo "task 1";
+echo "------------------------"
+echo ""
+echo ""
+echo "Install \"aptitude\" - program for package management";
+
+apt install --assume-yes aptitude
```

Comment:
task 1
------------------------


Install "aptitude" - program for package management
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package aptitude is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'aptitude' has no installation candidate
./1.sh: line 27: aptitude: command not found